Control coverage/Feature/VirtualMachineScaleSet.md

```diff
@@ -1,6 +1,16 @@
+| <b>ControlId:</b><br>Azure_VirtualMachineScaleSet_SI_Enable_Auto_OS_Upgrade<br><br><b>DisplayName:</b><br>Enable automatic OS image upgrade on Virtual Machine Scale Set.<br><br><b>Description: </b><br> Enable automatic OS image upgrade on Virtual Machine Scale Set. |<b> ARM API to list all the VMSS configurations under the specified subscription: </b> <br> /subscriptions/{subscriptionId}/providers<br>/Microsoft.Compute/virtualMachineScaleSets<br>?api-version=2019-07-01 <br><br><b>Properties:</b><br>properties/upgradePolicy/automaticOSUpgradePolicy/enableAutomaticOSUpgrade| <b>Scope: </b> All VMSS in a subscription<br><br><b>Config: </b>NA<br><br><b>Passed: </b><br>Automatic OS image upgrade is configured for VM Scale Set.<br><br><b>Failed: </b><br>Automatic OS image upgrade is not configured for VM Scale Set. |
+| <b>ControlId:</b><br>Azure_VirtualMachineScaleSet_SI_Latest_Model_Applied<br><br><b>DisplayName:</b><br>All VMs in VM Scale Set must be up-to-date with the latest scale set model.<br><br><b>Description: </b><br> All VMs in VM Scale Set must be up-to-date with the latest scale set model. |<b> ARM API to list all the VMSS configurations under the specified subscription: </b> <br> /subscriptions/{subscriptionId}/providers<br>/Microsoft.Compute/virtualMachineScaleSets<br>?api-version=2019-07-01 <br><br><b>Properties:</b><br>properties/upgradePolicy/mode<br><br><b> ARM API to get all the VM instances for the specified VMSS: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers<br>/Microsoft.Compute/virtualMachineScaleSets/{resourceName}/virtualMachines<br>?api-version=2019-07-01 <br><br><b>Properties:</b><br> instance/properties/latestModelApplied| <b>Scope: </b> All VMSS in a subscription<br><br><b>Config: </b>NA<br><br><b>Passed: </b><br>VMSS upgrade policy is not manual <br>or VMSS upgrade policy is manual but all the VM <br>instances are running on latest VM Scale Set model.<br><br><b>Failed: </b><br>VMSS upgrade policy is manual and any of the VM <br>instance is not running on latest VM Scale Set model. |
+| <b>ControlId:</b><br>Azure_VirtualMachineScaleSet_SI_Missing_OS_Patches<br><br><b>DisplayName:</b><br>System updates on virtual machine scale sets must be installed.<br><br><b>Description: </b><br> Virtual Machine Scale Set must have all the required OS patches installed. |<b> ARM API to fetch all the security assessment for the specified subscription: </b> <br> /subscriptions/{subscriptionId}/providers<br>/Microsoft.Security/assessments<br>?api-version=2020-01-01 <br><br><b>AssessmentName:</b><br>bd20bd91-aaf1-7f14-b6e4-866de2f43146| <b>Scope: </b> All VMSS in a subscription<br><br><b>Config: </b>NA<br><br><b>Passed: </b><br>ASC assessment is in healthy state.<br><br><b>Failed: </b><br>ASC assessment is in unhealthy state. |
```

Comment:
You may want to call out what fields are considered for the evaluation. For example, the status code from the assessment.